docker/gaffer-pyspark-notebook/.env

```diff
@@ -0,0 +1,9 @@
+ZOOKEEPER_VERSION=3.7.0
```

Comment:
Are these versions managed? Is that what https://github.com/gchq/gaffer-docker/pull/138 does?

Reply:
No, we could do with an automated way of making sure that all the version numbers in dockerfiles, docker-compose env files, kubernetes values files are consistent. #138 just tries to make sure that when the CI/CD tests are run, they test the container images that have been built, rather than pulling released container images from Docker Hub and testing them.

Reply:
Okay that makes sense, I have seen that problem before so #138 sounds useful.

I think perhaps in the future something like that would be nice, as currently the process of upgrading the Gaffer version is a find and replace job, but that's out of the scope of this so I will resolve.